content/html/concepts/elements/terms/style/style.md

```diff
@@ -0,0 +1,80 @@
+---
+Title: '<style>'
+Description: 'This element is used to apply a simple style sheet to an HTML document'
```

Comment:
```suggestion
Description: 'Applies CSS styles to an HTML document.'
```

For term entries, let's begin the `Description` with the verb of the first sentence for this entry (and don't forget punctuation).